operations/app/terraform/modules/function_app/apps.tf

```diff
@@ -20,35 +20,43 @@ module "pdi_function_app" {
-    FHIR_URL                              = "https://${var.resource_prefix}-fhir.azurehealthcareapis.com"
+    PRIVATE_KEY                            = "@Microsoft.KeyVault(SecretUri=https://${var.resource_prefix}-app-kv.vault.azure.net/secrets/PrivateKey)"
+    PRIVATE_KEY_PASSWORD                   = "@Microsoft.KeyVault(SecretUri=https://${var.resource_prefix}-app-kv.vault.azure.net/secrets/PrivateKeyPassword)"
+    AZURE_STORAGE_CONTAINER_NAME           = "bronze"
+    APPINSIGHTS_INSTRUMENTATIONKEY         = var.ai_instrumentation_key
+    APPLICATIONINSIGHTS_CONNECTION_STRING  = var.ai_connection_string
+    BUILD_FLAGS                            = "UseExpressBuild"
+    FUNCTIONS_WORKER_RUNTIME               = each.value.runtime
+    SCM_DO_BUILD_DURING_DEPLOYMENT         = each.value.SCM_DO_BUILD_DURING_DEPLOYMENT
+    VDHSFTPHostname                        = "vdhsftp.vdh.virginia.gov"
+    VDHSFTPPassword                        = "@Microsoft.KeyVault(SecretUri=https://${var.resource_prefix}-app-kv.vault.azure.net/secrets/VDHSFTPPassword)"
+    VDHSFTPUsername                        = "USDS_CDC"
+    XDG_CACHE_HOME                         = "/tmp/.cache"
+    WEBSITE_RUN_FROM_PACKAGE               = each.value.WEBSITE_RUN_FROM_PACKAGE
+    DATA_STORAGE_ACCOUNT                   = var.sa_data_name
+    ENABLE_ORYX_BUILD                      = each.value.ENABLE_ORYX_BUILD
+    "${each.value.mi_blobServiceName}"     = each.value.mi_blobServiceUri
+    "${each.value.mi_queueServiceName}"    = each.value.mi_queueServiceUri
+    "${each.value.mi_accountName}"         = each.value.mi_accountValue
+    fhir_url                               = each.value.fhir_url
+    "AzureWebJobs.convertToFhir.Disabled"  = each.value.AzureWebJobs_convertToFhir_Disabled
+    SMARTYSTREETS_AUTH_ID                  = "@Microsoft.KeyVault(SecretUri=https://${var.resource_prefix}-app-kv.vault.azure.net/secrets/SmartyStreetsAuthID)"
+    SMARTYSTREETS_AUTH_TOKEN               = "@Microsoft.KeyVault(SecretUri=https://${var.resource_prefix}-app-kv.vault.azure.net/secrets/SmartyStreetsAuthToken)"
+    CONTAINER_URL                          = "https://${var.resource_prefix}datasa.blob.core.windows.net/bronze"
+    INTAKE_CONTAINER_URL                   = "https://${var.resource_prefix}datasa.blob.core.windows.net/bronze"
+    INTAKE_CONTAINER_PREFIX                = "decrypted/valid-messages/"
+    OUTPUT_CONTAINER_PATH                  = "processed"
+    CSV_INPUT_PREFIX                       = each.value.CSV_INPUT_PREFIX
+    CSV_OUTPUT_PREFIX                      = each.value.CSV_OUTPUT_PREFIX
+    HASH_SALT                              = "@Microsoft.KeyVault(SecretUri=https://${var.resource_prefix}-app-kv.vault.azure.net/secrets/salt)"
+    FHIR_URL                               = "https://${var.resource_prefix}-fhir.azurehealthcareapis.com"
+    "AzureWebJobs.IntakePipeline.Disabled" = each.value.AzureWebJobs_IntakePipeline_Disabled
```

Comment:
Does this disable our `IntakePipeline` from running? I assume no, but would appreciate some insight here to help me understand what it's doing.

Reply:
Similar to above, I only want to match with the values you are currently using.
The current value is 0, so this will not disable it.
https://github.com/CDCgov/prime-public-health-data-infrastructure/pull/113/files#diff-0ffe207d8985f822098698b3dc16966f2f0182a7cb6d50591a8516cc915ae389R18